### PR TITLE
[MVS-459] Create endpoint for Appointment

### PR DESCRIPTION
**Pull request by axn9t9**
## :flipper: [JIRA Sprint Card]
https://mass-immunization-system.atlassian.net/browse/MVS-459

## :newspaper: [Work Description]
Defined template for incoming request data and add logic to structure the incoming request data into the FHIR standard.

## :vertical_traffic_light: [Testing/QA Notes]
- Run docker containers
- Create preliminary FHIR resources (referenced FHIR resources must exist)
- Send a POST request to /Appointment with the request data formatted according to template in models directory.
- Ensure the response data is the equivalent FHIR resource, with all of the provided information.

### CODE REVIEW

File: node_app/models/Appointment.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/*
+Appointment {
+  status : enum (proposed, pending, booked, arrived, fulfilled, cancelled, noshow, entered-in-error, checked-in, waitlist)
+  participant : [ 
+    {
+      type : "patient" // note that this type is an extensible
+      actor : string "Patient/id"
+    },
+    {
+      type : "practitioner" // note that this type is an extensible
```
Comment (dmcook5208) [inline node_app/models/Appointment.js:10]: We probably won't ever fill this in (the patient will just be registering for a bank of appointment openings for a given time slot), but it doesn't hurt to keep it.  I also think it's fine if you remove it.

File: node_app/models/Appointment.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          "type":"patient",
+          "actor":"Patient/example"
+        },
+        {
+          "type":"pratitioner",
+          "actor":"Practitioner/f201"
+        },
+        {
+          "type":"patient",
+          "actor":"Patient/102"
```
Comment (dmcook5208) [inline node_app/models/Appointment.js:35]: Is this a typo?  You have "patient" on lines 26/27 and 34/35

File: node_app/models/Appointment.js (hunk 1 above)
Comment (axn9t9) [reply node_app/models/Appointment.js:10]: You're right. I'm not sure how appointments will be scheduled for practitioners and patients so I'll hold off on this for now.

File: node_app/models/Appointment.js (hunk 2 above)
Comment (axn9t9) [reply node_app/models/Appointment.js]: Thanks. Fixed it.

**Review by dmcook5208 — APPROVED** [review]

**Pull request closed without merging**